python-lib/spacy_tokenizer.py

```diff
@@ -181,15 +182,19 @@ def _create_spacy_tokenizer(self, language: AnyStr) -> Language:
                 language == "th"
             ):  # PyThaiNLP requires a "data directory" even if nothing needs to be downloaded
                 os.environ["PYTHAINLP_DATA_DIR"] = mkdtemp()  # dummy temp directory
+                
             if language in SPACY_LANGUAGE_MODELS and self.use_models:
                 nlp = spacy.load(SPACY_LANGUAGE_MODELS[language])
-                nlp.remove_pipe("ner")
             else:
                 nlp = spacy.blank(
                     language
                 )  # spaCy language without models (https://spacy.io/usage/models)
+                
```

Comment:
Nitpick: it can be good to use blank lines to denote visual separation between steps in big methods. Cf. https://www.python.org/dev/peps/pep-0008/#blank-lines

However, since most of our current methods do not have blank lines, it does not look very consistent. I would either remove the blank lines, or add them everywhere.

Reply:
I removed them  baed249